### ISSUE TITLE
Unable to delete Task

### ISSUE THREAD

**Issue by brentdurksen** [ti-dev/Scrum-it#2]
I cannot find any documentation (in English at least) on how to delete a task from a Sprint view. It does not seem to be shown in the video tutorial either.

**Comment by dzhokn** [comment]
Unfortunately, there us no way to remove Task through the UI. The only way to do this is to access the Database and execute relevant SQL query.

Same approach could be applied for setting specific Task estimation (default is 8hrs)

**Comment by dzhokn** [comment]
I would use this thread to request DeleteTask feature.

**Comment by ti-dev** [comment]
mdzhokanov is right, it is not possible to delete a task via the GUI. I had a lot of discussions and the majority of the people told me that a task should never be deleted within a sprint, so it is a "Scrum thing".
But of course, it is very easy to implement it.

Only a UI action is needed which call the service which will perform the database delete action on the selected task. Also the burn down chart have to be updated.

**Comment by dzhokn** [comment]
Not removing tasks should be a convention within the Scream team. But disabling that feature in the application is nonsense. What should I do if I mistakenly click twice on the NewTask button?

DeleteTask feature is a mandatory functionality in my opinion.

**Comment by ti-dev** [comment]
Yes, it is our definition in the company, so we did it in our way. Feel free to change it.
